README.md

```diff
@@ -140,6 +140,9 @@ auto_top_growth_df.ai.verify("expect sales change percentage to be between -100
 > result: True
 
 ### UDF Generation
+
+#### Example 1: Compute expression from columns
```

Comment:
@SemyonSinchenko sorry but I would prefer keeping the README simple.

> @SemyonSinchenko Again, thanks for working on this. Shall we also mention this new notebook in https://github.com/pyspark-ai/pyspark-ai/blob/master/docs/udf_generation.md?

I meant to mention the new examples or notebook link in the `udf_generation.md`.

Reply:
Done